src/helm/benchmark/static/schema.yaml

```diff
@@ -474,6 +474,48 @@ models:
     num_parameters: 70000000000
     release_date: 2023-07-18
 
+  # Stanford
```

Comment:
Why do these show up as diffs in this PR?

Reply:
Ugh, not sure why this bad merge happened. I rebased and fixed this.